### PR TITLE
S2t6 frontend rate toppings page

### PR DESCRIPTION
**Pull request by amyas-gilbert**

### CODE REVIEW

**Comment by chloe-smart** [comment]
In terms of styling, I think it looks really good. But I'm wondering if the black is a bit jarring with the rest of the colour pallet? Maybe the h2 could be the same dark green as the h1 and the VS could be in the red (meat colour)?

**Comment by chloe-smart** [comment]
Also I think the VS on the mobile version is quite small and should be a bit bigger, and maybe the cards on mobile could also be slightly bigger as the text on the buttons becomes quite small

File: frontend/src/RateToppingsPage/RateToppingsPage.scss [hunk 1]
```diff
@@ -0,0 +1,46 @@
+@import '../styles/colors';
+@import '../styles/_fonts';
+
+.page {
+    background-color: #F1DDB8;
```
Comment (chloe-smart) [inline frontend/src/RateToppingsPage/RateToppingsPage.scss:5]: maybe use the scss background-color variable here?

**Comment by chloe-smart** [comment]
Currently when the page reloads it shows an empty button and an empty image - might be better to display 'loading' or something nice

File: frontend/src/RateToppingsPage/RateToppingsPage.tsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
+// imports
+import { useState, useEffect, ReactElement } from "react";
+import { JsxElement } from "typescript";
+import './RateToppingsPage.scss'
+import RateToppingCard from "../RateToppingCard/RateToppingCard";
+import { Topping } from "../RateToppingCard/RateToppingCard"
+
+const RateToppingsPage = (): JSX.Element => {
+
+    const dummyTopping: Topping = {
```
Comment (chloe-smart) [inline frontend/src/RateToppingsPage/RateToppingsPage.tsx:10]: maybe rename 'initial empty topping' or something

File: frontend/src/RateToppingsPage/RateToppingsPage.tsx [hunk 3]
```diff
@@ -0,0 +1,56 @@
+// imports
+import { useState, useEffect, ReactElement } from "react";
```
Comment (chloe-smart) [inline frontend/src/RateToppingsPage/RateToppingsPage.tsx:2]: why do you need ReactElement? just curious

**Comment by amyas-gilbert** [comment]
changed colours as seen and added loading text before images appear

**Review by chloe-smart — APPROVED** [review]

**Pull request merged**